### PR TITLE
Use `contextlib.suppress` instead of `try...except`

### PR DESCRIPTION
**Pull request by Uxio0**

### CODE REVIEW

File: safe_transaction_service/utils/tests/test_tasks.py [hunk 1]
```diff
@@ -1,16 +1,34 @@
+from unittest import mock
```
Comment (fmrsabino) [inline safe_transaction_service/utils/tests/test_tasks.py:1]: What is the context for these changes? 😅

**Review by fmrsabino — COMMENTED** [review]
Looks good to me. I didn't know about `contextlib.suppress`.

There are some other changes that I'm not sure that they are related with the PR 🤔

File: safe_transaction_service/utils/tests/test_tasks.py (hunk 1 above)
Comment (Uxio0) [reply safe_transaction_service/utils/tests/test_tasks.py:1]: Oh that's right, those changes were supposed to go on the increase coverage PR. I realised some of the tasks were not tested at all

Comment (Uxio0) [reply safe_transaction_service/utils/tests/test_tasks.py]: Removed

**Review by fmrsabino — APPROVED** [review]

**Pull request merged**